pkg/accountmanager/manager_test.go

```diff
@@ -0,0 +1,193 @@
+					"credentials": []byte(credential),
+				},
+			}
+			ces = &v1alpha1.CloudEntitySelector{
+				ObjectMeta: v1.ObjectMeta{
+					Name:      testCesNamespacedName.Name,
+					Namespace: testCesNamespacedName.Namespace,
+					OwnerReferences: []v1.OwnerReference{
+						{
+							APIVersion: "crd.cloud.antrea.io/crdv1alpha1",
+							Kind:       "CloudProviderAccount",
+							Name:       testAccountNamespacedName.Name,
+						},
+					},
+				},
+				Spec: v1alpha1.CloudEntitySelectorSpec{
+					AccountName: testAccountNamespacedName.Name,
+					VMSelector: []v1alpha1.VirtualMachineSelector{
+						{
+							VpcMatch: &v1alpha1.EntityMatch{
+								MatchID: "xyzq",
+							},
+						},
+					},
+				},
+			}
+			accountManager.ConfigureAccountManager()
+			accountCloudType, err = util.GetAccountProviderType(account)
+			Expect(err).ShouldNot(HaveOccurred())
+		})
+		It("Add/Remove Account", func() {
+
+			//Secret is not created.
+			err := accountManager.AddAccount(&testAccountNamespacedName, accountCloudType, account)
+			Expect(err).Should(HaveOccurred())
+
+			//Invalid account cloud type.
+			err = accountManager.AddAccount(&testAccountNamespacedName, "", account)
+			Expect(err).Should(HaveOccurred())
+
+			//Valid add account.
+			_ = fakeClient.Create(context.Background(), secret)
+			err = accountManager.AddAccount(&testAccountNamespacedName, accountCloudType, account)
+			Expect(err).ShouldNot(HaveOccurred())
+
+			//Verify whether the poller is restarted.
+			err = accountManager.AddAccount(&testAccountNamespacedName, accountCloudType, account)
+			Expect(err).ShouldNot(HaveOccurred())
+
+			//Delete the account.
+			err = accountManager.RemoveAccount(&testAccountNamespacedName)
+			Expect(err).ShouldNot(HaveOccurred())
+
+		})
+		It("Add/Remove Account Poller", func() {
+			// Add account poller
+			cloudProviderType := accountManager.addAccountProviderType(&testAccountNamespacedName, accountCloudType)
+			cloudInterface, err := cloudprovider.GetCloudInterface(cloudProviderType)
+			Expect(err).ShouldNot(HaveOccurred())
+			_, exists := accountManager.addAccountPoller(cloudInterface, &testAccountNamespacedName, account)
+			Expect(exists).To(Equal(false))
+
+			// Already account poller exists
+			_, exists = accountManager.addAccountPoller(cloudInterface, &testAccountNamespacedName, account)
+			Expect(exists).To(Equal(true))
+
+			// Remove account poller.
+			err = accountManager.removeAccountPoller(&testAccountNamespacedName)
+			Expect(err).ShouldNot(HaveOccurred())
+
+			// Account poller not found.
+			err = accountManager.removeAccountPoller(&testAccountNamespacedName)
+			Expect(err).Should(HaveOccurred())
+		})
+		It("Add/Remove Resource Filters Account ", func() {
+
+			// Account is not added so throws error.
+			_, err := accountManager.AddResourceFiltersToAccount(&testAccountNamespacedName, &testCesNamespacedName, ces)
+			Expect(err).Should(HaveOccurred())
+
+			// Invalid account namespace.
```

Comment:
Since its account namespaced name.
```suggestion
			// Invalid account namespaced name.
```

Reply:
Solved.